src/coreclr/tools/Common/Compiler/HardwareIntrinsicHelpers.cs

```diff
@@ -273,6 +276,8 @@ public static int FromInstructionSet(InstructionSet instructionSet)
                     InstructionSet.ARM64_Atomics => Atomics,
                     InstructionSet.ARM64_Rcpc => Rcpc,
                     InstructionSet.ARM64_Rcpc2 => Rcpc2,
+                    InstructionSet.ARM64_Sve => Sve,
```

Comment:
As mentioned elsewhere, IMO, this should 
```c#
InstructionSet.Arm64_Sve => Sve,
InstructionSet.Arm64_Sve2 => Sve2,
```

Reply:
This PR isn't adding Sve. These entries are handling `Sve` and the nested class `Sve.Arm64`, which is consistent with all other ISAs we've added so far.

Reply:
yep, I was suggesting to do `Sve2` as well in this PR.